spec/system/avo/trix_field_spec.rb

```diff
@@ -31,11 +31,17 @@
 
         save
 
-        click_on "Show content"
-
```

Comment:
In all instances where `click_on "Show content"` was removed, update the test logic to use `click_on "More content"` instead.

Since "More content" only appears when there is additional content, add extra text to `trix_post_body`.

It's essential to keep each test case focused on its original purpose. Here, we only need to add more content, and the test should confirm that the entire content displays correctly after clicking "More content."

Reply:
the js controller checks if the body has more then one line, so if it is not then More content button will not appear